Makefile

```diff
@@ -127,6 +132,10 @@ eslint-branch: node_modules/eslint node_modules/eslint-plugin-react node_modules
 mixedindentlint: node_modules/mixedindentlint
 	@echo "$(JS_FILES)\n$(SASS_FILES)" | xargs $(NODE_BIN)/mixedindentlint --ignore-comments --exclude="client/config/index.js"
 
+# Ensure that default config values exist in _shared.json
+config-defaults-lint: $(CONFIG_FILES)
+	@if [[ ! -z "$$(FORCE_COLOR=1 node server/config/validate-config-keys.js)" ]]; then exit 1; fi;
```

Comment:
Can someone review the use of `FORCE_COLOR` here and the whole line altogether? I copied this pattern from #10234